lms/djangoapps/grades/signals/handlers.py

```diff
@@ -179,6 +179,11 @@ def enqueue_subsection_update(sender, **kwargs):  # pylint: disable=unused-argum
     enqueueing a subsection update operation to occur asynchronously.
     """
     _emit_problem_submitted_event(kwargs)
+
+    if kwargs['score_db_table'] == ScoreDatabaseTableEnum.submissions and kwargs['weighted_possible'] == 0:
+        # This scenario is known to not succeed, see TNL-6559 for details.
+        return
```

Comment:
Should I log the would-have-been task's details here? We're already emitting the event, so I'm not sure what logging is needed.